src/components/TokenCard/TokenCard.js

```diff
@@ -106,6 +142,10 @@ export default function TokenCard({ showRedirectModal, redirectPopupTimestamp })
             </div>
           </div>
         </div>
+        <div className="Home-token-card-option-apr">
```

Comment:
let's add `aprRangeText && ` or more granular checks otherwise it would look broken if `!arbApr || !arbIncentiveApr || !avaxApr || !avaxIncentiveApr` happens